mica/stats/acq_stats.py

```diff
@@ -441,6 +448,43 @@ def _get_obsids_to_update(check_missing=False):
     return obsids
 
 
+def warn_on_acq_anom(acq, starcheck, to_addr):
```

Comment:
It will also help if you adhere to the [recommended idiom](https://occweb.cfa.harvard.edu/twiki/bin/view/Aspect/SkaPython#Variable_and_Method_Names) of using a plural for list-like things where it is not obvious.  For instance `acqs` would give me a hint right away that this is not one acq star but instead a list of table of stars.  Likewise `emails` would hint that is a list of email addresses.